sphinx_gallery/gen_rst.py

```diff
@@ -902,9 +927,14 @@ def rst_blocks(script_blocks, output_blocks, file_conf, gallery_conf):
     # A simple example has two blocks: one for the
     # example introduction/explanation and one for the code
     is_example_notebook_like = len(script_blocks) > 2
-    example_rst = u""  # there can be unicode content
-    for (blabel, bcontent, lineno), code_output in \
-            zip(script_blocks, output_blocks):
+    example_rst = ""
+    for bi, ((blabel, bcontent, lineno), code_output) in \
```

Comment:
why `bi` .. ? block index?

Reply:
yes I just like to avoid `i` and `j` so I usually prefix with something at least somewhat meaningful but still short